lib/assesments/assessment_2/fi_a2_muhammad_bilal/fi_a2_mbilal_homescreen/widgets/fi_a2_mbilal_homecontainer.dart

```diff
@@ -0,0 +1,125 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class BilalHomeContainer extends StatelessWidget {
+  const BilalHomeContainer({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 27,top: 15,right: 26),
+      child: Container(
+        width:double.infinity,
+        height: 440,
+        decoration: BoxDecoration(
+          borderRadius:BorderRadius.circular(20),
+          color: Color(0xFFFCFCFC),
+          boxShadow: [
+            BoxShadow(
+              color: Colors.grey.withOpacity(0.5),
+              spreadRadius:3,
+              blurRadius:4,
+              offset: Offset(0, 3), // changes position of shadow
+            ),
+          ],
+        ),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Padding(
+              padding:const EdgeInsets.only(top:10),
+              child: ListTile(
+                leading:Container(
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(40),
+                    border: Border.all(
+                      width: 3,
+                      color: Color(0xFFFA9884),
+                    ),
+
+                  ),
+                  child: CircleAvatar(
+                      backgroundColor: Colors.white,
+                      radius: 25,
+                      child: Image.asset('assets/fi_a2_mbilal_assets/man3.png',width: 45,height: 45,)),
```

Comment:
Please add trailing commas and format the code

Reply:
added